### PR TITLE
I expect the previous code never reached the else case and only worked with Dart1 type checks

### PR DESCRIPTION
**Pull request by jacob314**

### CODE REVIEW

File: lib/codemirror.dart [hunk 1]
```diff
@@ -1124,7 +1124,7 @@ class TextMarker extends ProxyHolder {
     if (result is! JsObject) return null;
 
     try {
-      if (result is Map) {
+      if (result is JsObject && result.hasProperty('from')) {
```
Comment (devoncarew) [inline lib/codemirror.dart]: is the '`result is JsObject`' needed to satisfy the analyzer? We're already doing an `is!` check above

File: pubspec.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # license that can be found in the LICENSE file.
 
 name: codemirror
-version: 0.5.9+5.48.4
+version: 0.5.10
```
Comment (devoncarew) [inline pubspec.yaml]: perhaps `0.5.10+5.48.4`; `5.48.4` is the JS code mirror version

File: lib/codemirror.dart (hunk 1 above)
Comment (jacob314) [reply lib/codemirror.dart]: Interesting. It is needed to get the analyzer to suggest the right completions on result but the analyzer doesn't seem to complain without it.

File: pubspec.yaml (hunk 2 above)
Comment (jacob314) [reply pubspec.yaml]: Ah that is what that naming scheme was. We'll start using it for js interop based libraries in devtools as well.

**Pull request merged**